### ISSUE TITLE
Trying to archive getting this error : Command CodeSign failed with a nonzero exit code

### ISSUE THREAD

**Issue by eraldix** [inspireui/support#4320]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: Im trying to archive project and i'm getting **Command CodeSign failed with a nonzero exit code** over one signal notification
+ Product & version: Multivendor Latest
+ Flutter (or React Native) version: Flutter
+ Testing Device/Simulator: xcode
+ Screenshot issues (drag the file to attach here): 
<img width="1680" alt="Screenshot 2020-07-22 at 6 34 43 PM" src="https://user-images.githubusercontent.com/44481659/88203847-b84e3200-cc4a-11ea-8a26-8c09427586da.png">

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by ameenbadri7** [comment]
Thank you for the feedback.

Please help us to submit this form and fill all required info to continue with the support - http://verify.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.

We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission. 

Thank you for your time!

**Comment by eraldix** [comment]
> Thank you for the feedback.
> 
> Please help us to submit this form and fill all required info to continue with the support - http://verify.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.
> 
> We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission.
> 
> Thank you for your time!

Please check, I have done that already

**Comment by eraldix** [comment]
**OneSignalNotificationServiceExtension.appex: errSecInternalComponent
    Command CodeSign failed with a nonzero exit code**
This is what I get when i **run flutter build ios**

**Comment by minhcasi** [comment]
@eraldix - try to open the XCode and check this option https://tppr.me/1uwe

**Comment by eraldix** [comment]
> @eraldix - try to open the XCode and check this option https://tppr.me/1uwe - the team account should match with your own Apple account.
> 
> Let us know if you have any questions. Please note that we will not support answering or support technical issues for the regular license as per support policies - https://inspireui.com/support-policies/ - Hope you enjoy the product and stay safe 😊

When you change the support policies you change it for the new people that come in , not for the old ones that had another support policies when they bought, don't make give you low reviews on codecanyon and everywhere else.

**Comment by eraldix** [comment]
Tried that, also tried cleaning pods, cleaning xcode workspace, restarting pc, lock and unlock login on keychain and nothing has happened. this is what i get when I run flutter build on android studio...

  /Users/pentazon/Downloads/JOZON/source/build/ios/Release-iphoneos/OneSignalNotificationServiceExtension.appex: replacing existing signature
    /Users/pentazon/Downloads/JOZON/source/build/ios/Release-iphoneos/OneSignalNotificationServiceExtension.appex: errSecInternalComponent
    Command CodeSign failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

**Comment by minhcasi** [comment]
@eraldix - try to read the detail setting for onesignal form this link to set up your own bundle id with onesignal, make sure you have purchased the Apple Developer account - https://documentation.onesignal.com/docs/flutter-sdk-setup
